infra/main.bicep

```diff
@@ -69,6 +69,9 @@ param azureSearchTitleColumn string = 'title'
 @description('Url column')
 param azureSearchUrlColumn string = 'url'
 
+@description('Use Azure Search Integrated Vectorization')
```

Comment:
```suggestion
@description('Use Azure Search Integrated Vectorization (Not yet implemented')
```

We should probably have some kind of note that this isn't available yet, to not cause confusion

Reply:
Great point